project/src/index.js

```diff
@@ -2,9 +2,17 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './components/app/app';
 
+const OFFERS_COUNT = 9;
+const createOffers = (offersCount) => {
+  const offers = new Array(offersCount).fill('');
+  return offers.map((elem,index) => {
+    elem = {id: index};
+    return elem;
+  });
+};
 
 ReactDOM.render(
   <React.StrictMode>
-    <App />
+    <App  offers={createOffers(OFFERS_COUNT)}/>
```

Comment:
Понятно что это временно но я бы лучше передавал сюда уже результат выполнения функции `const offers = createOffers(OFFERS_COUNT);`
```suggestion
    <App  offers={offers}/>
```
И надо убрать 1 пробел после App